examples/machine-learning-pipeline/business_unit_3/development/main.tf

```diff
@@ -22,5 +22,6 @@ module "base_env" {
   environment_code = local.environment_code
   business_code    = local.business_code
   project_id       = local.machine_learning_project_id
+  project_number   = local.project_number
```

Comment:
not sure if all required variables are in this code block, like production_project_id